sepal_ui/mapping/zoom_control.py

```diff
@@ -0,0 +1,69 @@
+"""
+Customized ``Control`` to zoom in and out on the map.
+"""
+from typing import Optional
+
+from ipyleaflet import Map, WidgetControl
+
+from sepal_ui import sepalwidgets as sw
+from sepal_ui.mapping.map_btn import MapBtn
+
+
+class ZoomControl(WidgetControl):
+
+    plus: Optional[MapBtn] = None
+    "The plus btn"
+
+    minus: Optional[MapBtn] = None
+    "the minus btn"
+
+    m: Optional[Map] = None
+    "the map to manipulate"
+
+    def __init__(self, m: Map, **kwargs) -> None:
+        """
+        Customized ``Control`` to zoom in and out on the map.
+
+        Replace the built-in zoom control of ipyleaflet to match the theme of sepal-ui based applications. It is by default positioned in the top-right corner
+
+        Args:
+            m: the map to manipulate
+            kwargs: any ``ipyleaflet.widgetControl`` arguments
+        """
+        # init the map
+        self.m = m
+
+        # create the 2 btns
+        self.plus = MapBtn("fa-solid fa-plus", _metadata={"step": 1})
+        self.minus = MapBtn("fa-solid fa-minus", _metadata={"step": -1})
```

Comment:
at some point you recommended use `attributes` parameter instead of `_metadata`... since attributes will always be a dictionary, in this case it's not harmful, but for consistency could we change it to attributes? and if not, in which cases you'd prefer one over the other?

Reply:
sure thing